CHANGES.txt

```diff
@@ -86,6 +80,16 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
         macro which is not located in a state TABLE.
       * Cleanup CPP expressions before evaluating (strip comments, carriage returns)
 
+  From Daniel Moody:
+    - Add no_progress (-Q) option as a set-able option. However, setting it in the 
+      SConstruct/SConscript will still cause "scons: Reading SConscript files ..." to be
+      printed, since the option is not set when the build scripts first get read.
+    - Added check for SONAME in environment to setup symlinks correctly (Github Issue #3246)
```

Comment:
Remove this. It's not in main, it's from your other PR right?

Reply:
I noticed the NOTE about alphabetical order, and moved all my changes to the correct spot for the latest release